sensors/utils/utils.go

```diff
@@ -0,0 +1,42 @@
+// Package utils contains helper functions for the sensor implementations.
+package utils
+
+import (
+	"context"
+	"image"
+
+	"github.com/edaniels/gostream"
+	"github.com/pkg/errors"
+	"go.viam.com/rdk/components/camera"
+	"go.viam.com/rdk/rimage"
+	"go.viam.com/rdk/utils"
+)
+
+// GetPNGImage first attempts to get a lazy PNG image. If the image is not a lazy PNG, the
+// function expects it to be an YCBCR image. If it is neither a lazy PNG nor an YCBCR, the
+// function errors out.
+func GetPNGImage(ctx context.Context, cam camera.Camera) ([]byte, func(), error) {
+	// We will hint that we want a PNG.
+	// The Camera service server implementation in RDK respects this; others may not.
+	readImgCtx := gostream.WithMIMETypeHint(ctx, utils.WithLazyMIMEType(utils.MimeTypePNG))
+	img, release, err := camera.ReadImage(readImgCtx, cam)
+	if err != nil {
+		return nil, nil, err
+	}
```

Comment:
Are we concerned about some program ignoring these errors and continuing to read images? Would that cause a memory leak?

Reply:
Actually, that is a really good point. I wonder if we should call that release function before returning an error...

Reply:
`release` is always `nil` when `err != nil`. I'll add a comment to make this clear.

Reply:
I am always returning the `release` function now.